types/web-element.d.ts

```diff
@@ -187,7 +187,12 @@ export interface ScopedElement extends Element, PromiseLike<WebElement> {
 
   rightClick(): Promise<WebElement>;
 
-  waitUntil(signalOrOptions: WaitUntilActions | WaitUntilOptions, waitOptions?: WaitUntilOptions): Promise<WebElement>;
+  isEnabled(): Promise<string | null>;
```

Comment:
Can you please revert the unrelated changes? Also, can you add a test for types? You can use this as a reference: https://github.com/nightwatchjs/nightwatch/blob/main/types/tests/webElement.test-d.ts